apps/frontend/src/libs/helpers/handle-click-outside/handle-click-outside.helper.ts

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useRef } from "~/libs/hooks/hooks.js";
+
+const handleClickOutside = <T extends HTMLElement>(
```

Comment:
```suggestion
const useClickOutside = <T extends HTMLElement>(
```
Let's make it a hook